platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -52,7 +52,9 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+## Improved error handling for toolchain resolvers
 
+When attempting to download Java toolchains from the set of configured resolvers errors will be better handled now and all resolvers will be tried. 
```

Comment:
```suggestion
When attempting to download Java toolchains from the configured resolvers, errors will be better handled now, and all resolvers will be tried. 
```

Reply:
- Is it possible to add a little bit about why we did this and who it helps?
- Is it possible to add a screenshot or snippet of the error?

Reply:
I don't think a screenshot makes sense, but I've added much more detail to the description. Please check.